packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -316,14 +337,18 @@ You might notice that the URL has your Okta `dev-*` domain, rather than your cus
 To fix this, update your authorization server to use your custom domain:
 
 1. Sign in to your Okta account and go to **Security** > **API** > **Authorization Servers**.
+
 1. Locate the default** custom authorization server, and then click **Edit**.
+
 1. In the **Settings** section, click **Edit**.
+
 1. Change the **Issuer** to **Dynamic (based on request domain)**.
+
 1. Make another request to `./well-known/openid-configuration`. Your custom domain is returned.
 
 ## About custom email addresses
 
-A custom email address allows you to present a branded experience to your end users. Emails that Okta sends to your end users appears to come from your custom email address instead of `noreply@okta.com`. You can switch to a different custom email address or revert to the default Okta domain, but you can use only one email domain at a time.
+A custom email address allows you to present a branded experience to your end users. Emails that Okta sends to your end users appears to come from your custom email address instead of `noreply@okta.com`. You can switch to a different custom email address or revert to the default Okta domain. But you can use only one email domain at a time.
```

Comment:
```suggestion
A custom email address allows you to present a branded experience to your end users. Emails that Okta sends to your end users appear to come from your custom email address instead of `noreply@okta.com`. You can switch to a different custom email address or revert to the default Okta domain, but you can use only one email domain at a time.
```